core/tx_pool.go

```diff
@@ -1441,10 +1442,6 @@ func (pool *TxPool) reset(oldHead, newHead *types.Header) {
 
 	// when we reset txPool we should explicitly check if fee struct for min base fee has changed
 	// so that we can correctly drop txs with < minBaseFee from tx pool.
-	// TODO: this should be checking IsSubnetEVM since we also support minimumFee for SubnetEVM
```

Comment:
We now require `SubnetEVMTimestamp` at genesis: https://github.com/ava-labs/subnet-evm/pull/419/files